company.el

```diff
@@ -118,9 +118,13 @@ attention to case differences."
   "Face used for the selection in the tooltip.")
 
 (defface company-tooltip-search
-  '((default :inherit company-tooltip-selection))
+  '((default :inherit highlight))
   "Face used for the search string in the tooltip.")
 
+(defface company-tooltip-search-selection
```

Comment:
I was keeping it consistent with `company-tooltip-selection-annotation-selection` and all the other faces.